src/main/java/Command.java

```diff
@@ -72,6 +72,10 @@ public void execute(TaskList tasks, Ui ui, Storage storage) {
             tasks.delete(deletionNumber);
             ui.printDelete(toDelete);
             break;
+        case "find":
```

Comment:
Each case can be extracted out into separate functions or sub-classes for readability.